bench/report/rusage.ml

```diff
@@ -2,4 +2,8 @@ type t =
   { clock : float
   ; utime : float
   ; stime : float
+  ; maxrss : int64
   }
+
+let pp fmt { clock; utime; stime; maxrss } =
+  Fmt.pf fmt "%g %g %g %Ld" clock utime stime maxrss
```

Comment:
While we're at it, do you think we could use `%4.2f` instead of `%g` ?

Reply:
Ofc! Will change this